### DISCUSSION TITLE
recursive search for dns/dnssec-detection.yaml

### DISCUSSION THREAD

**Discussion by acbekoac** [projectdiscovery/nuclei-templates#7495] [Q&A]
the  dnssec-detection template finds "In DS" records for "-u subdomain.domain.tld" but not if "-u www.subdomain.domain.tld" is used

**Comment by ehsandeep** [comment]
@acbekoac please include an example host where DS records exists for `www.subdomain.domain.tld` and nuclei failed to display.
